src/styles/nav.scss

```diff
@@ -0,0 +1,57 @@
+@import "variables.scss";
+
+.nav {
+  &__list {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+    display: flex;
+  }
+
+  &__item {
+    margin-right: 20px;
+    display: flex;
+  }
+
+  &__link {
+    display: inline-flex;
+    align-items: center;
+    min-height: 60px;
+
+    text-transform: uppercase;
+    text-decoration: none;
+    color: $color-dark;
+    font-size: 12px;
+    cursor: pointer;
+    transition: color 0.3s;
+    font-weight: 500;
+  }
+
+  &__item:last-child {
+    margin-right: 0;
+  }
```

Comment:
To prevent code copied in this case, you can write rules  `&__item:not(:last-child)`